.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,756 @@
+First of all - thank you for your interest in contributing to Kirby! 🥳
+<!-- Early on show that we are here to help if necessary and to educate people in how to get help -->
+If you are looking for help or at any point have questions, check out "[Help!: i have a question related to kirby](SUPPORT.md)". 
+
```

Comment:
When everything else is done, we should throw in a "How to use this document" section here.